### ISSUE TITLE
[Bug] => Python 3 treats TabErrors as syntax errors

### ISSUE THREAD

**Issue by cclauss** [0xInfection/XSRFProbe#3]
[flake8](http://flake8.pycqa.org) testing of https://github.com/theInfectedDrake/XSRFProbe on Python 3.7.0

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./core/xsrf_main.py:58:47: E999 TabError: inconsistent use of tabs and spaces in indentation
			soup=crawler.process(web) # start the parser
                                              ^
1     E999 TabError: inconsistent use of tabs and spaces in indentation
1
```

**Comment by 0xInfection** [comment]
For now I would prefer to keep XSRFProbe to [Python 2](https://github.com/theInfectedDrake/XSRFProbe#xsrf-probe----). I would consider doing a full code refactor sometime later to suit it for Python 3.x versions, but rn, I am in the middle of my exams and am running too short on time.

Feel free to contribute to this project. Any help in fixing this would be highly appreciated. :)

**Comment by 0xInfection** [comment]
Looks like we need to run the [reindent.py](https://github.com/python/cpython/blob/master/Tools/scripts/reindent.py) over this project too. 😀👌

**Comment by 0xInfection** [comment]
This gets fixed in #6. :)

Done!

**Comment by theInfectedDrake** [comment]
Migration successful, hence this gets fixed :)
